scimma_admin/hopskotch_auth/templates/hopskotch_auth/edit_credential.html

```diff
@@ -4,7 +4,7 @@
 {% block page-header %}Credential Management{% endblock %}
 
 {% block page-body %}
-        <h2>Credential {{ cred.username }}</h2>
+        <h2>Credential {{ cred.username }} ({{ cred.nickname }})</h2>
```

Comment:
This will look pretty strange if the nickname isn't set to a custom value, like `swnelson-05afe782 (swnelson-05afe782)`.

Reply:
I might be able to hide this if the nickname is the same as the username.